### PR TITLE
Added Pastebin command

### PR DESCRIPTION
**Pull request by etp12**
## Overview

This was a feature mentioned under "Issues" that I wanted to give a shot. Wasn't marked "help wanted" so if you don't want to use it, it's fine. 

This uses the Pastebin Api (developer api key must be set in config.js) to allow users to create Pastebin pastes using a "!paste {text}" command and returns a url to their paste. I did my best to stick to ES6 and the procedures set up in other commands.

It passes the test I made, but this was my first encounter with Nock, so it should be double-checked. I used the other test files as a template. 

Also added command info under the English "help" section.

## Screenshots

![Screenshot 1]
(https://i.gyazo.com/a6771d8c9f966dd1b9a0f628af074a3d.png)

![Screenshot 2]
(https://i.gyazo.com/740d319222fccec520e22cb60fe15314.png)

### CODE REVIEW

File: lib/help/english.js [hunk 1]
```diff
@@ -54,7 +54,9 @@ export default {
 **\`!wolfram\`** \`query\`
 		Query Wolfram Alpha for almost anything
 **\`!videocall\`**
-    Start a one click video call or screenshare directly on Appear.in. Mention users to make it private.`,
+    Start a one click video call or screenshare directly on Appear.in. Mention users to make it private.
+**\`!paste\`**\`text\`
```
Comment (dustinblackman) [inline lib/help/english.js]: Were trying to keep the help section alphabetical. Mind moving it up a bit, please? :)

File: lib/pastebin.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    method: 'POST',
+    form: {
+      api_option: 'paste',
+      api_paste_code: paste,
+      api_dev_key: nconf.get('PASTEBIN_KEY')
+    }
+  };
+
+  request(options)
+  .then(R.prop('body'))
```
Comment (dustinblackman) [inline lib/pastebin.js]: The promise chains after `request(options)` are usually indented.

**Comment by dustinblackman** [comment]
@etp12 This is great, thanks. Just two very minor comments. Give me a ping once they're fixed and I'll be happy to merge!

File: lib/pastebin.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  .then(text => bot.sendMessage(msg.channel, text))
+  .done();
+}
+
+export default {
+  paste: makePaste,
+  pastebin: makePaste,
+  makePaste: makePaste
```
Comment (dustinblackman) [inline lib/pastebin.js]: `make-paste`

File: lib/pastebin.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      api_option: 'paste',
+      api_paste_code: paste,
+      api_dev_key: nconf.get('PASTEBIN_KEY')
+    }
+  };
+
+  request(options)
+  .then(R.prop('body'))
+  .then(text => bot.sendMessage(msg.channel, text))
+  .done();
```
Comment (dustinblackman) [inline lib/pastebin.js]: `done` doesn't exist. Adds this to the top of the file;
`import sentry from './config/sentry';`

And replace `done` with this.
```
  .catch(err => {
      sentry.captureError(err);
      bot.sendMessage(msg.channel, `Error: ${err.message}`);
    });
```

**Comment by dustinblackman** [comment]
@etp12 This is great, thanks. Just four very minor comments. Give me a ping once they're fixed and I'll be happy to merge!

File: lib/pastebin.js (hunk 3 above)
Comment (etp12) [inline lib/pastebin.js]: I'm not sure what you mean here, do you want to add a "make-paste: makePaste" or change the name of the function?

Comment (dustinblackman) [inline lib/pastebin.js]: Just the name of the alias (so the object key). `'make-paste': makePaste`

**Comment by etp12** [comment]
@dustinblackman Let me know if these changes look good!

**Comment by dustinblackman** [comment]
Perfect, thanks again!

**Pull request merged**